src/net.c

```diff
@@ -1557,6 +1569,8 @@ handle_locally(coap_context_t *context __attribute__ ((unused)),
 handle_locally(coap_context_t *context, coap_queue_t *node) {
 #endif /* GCC */
   /* this function can be used to check if node->pdu is really for us */
+  (void)context;
```

Comment:
I can remove the conditional, but is this function really needed, since it always returns 1, regardless of the platform and arguments?